Azure-Samples/modern-data-warehouse-dataops
# Update ddo_transform python files and tests

# Type of PR


- Code changes
- Test changes
- Test Data changes

## Purpose

- Upload ddo_transform python file to fabric environment
- Update tests for ddo_transform python file

## Does this introduce a breaking change? If yes, details on what can break
No

## Author pre-publish checklist


## Validation steps
- Execute tests on ddo_transform python file to ensure it is working as expected. Tests files are placed under `modern-data-warehouse-dataops/e2e_samples/fabric_dataops_sample/src/tests`.
- Execute deploy.sh command and see if ddo_transform python file is uploaded to fabric environment

## Issues Closed or Referenced
- Closes #987
